src/app/home/index.njk

```diff
@@ -116,8 +113,8 @@
                 </td>
                 <td class="govuk-table__cell">{{ gar.departureDate }}</td>
                 <td class="govuk-table__cell">{{ gar.departureTime }}</td>
-                <td class="govuk-table__cell">{{ gar.departurePort | upper }}</td>
-                <td class="govuk-table__cell">{{ gar.arrivalPort | upper }}</td>
+                <td class="govuk-table__cell">{{ __('value_port_code_zzzz') if gar.departurePort == 'YYYY' else gar.departurePort | upper }}</td>
```

Comment:
Set the label to be "N/A" if not port. More for consistency as 'YYYY' nor 'ZZZZ' are referenced in the application itself anymore.